config/owasp/suppressions.xml

```diff
@@ -36,6 +36,30 @@
     <cve>CVE-2023-34034</cve>
     <cve>CVE-2023-20862</cve>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+     tomcat-embed-core-10.1.12.jar ]]></notes>
+    <cve>CVE-2023-44487</cve>
+    <cve>CVE-2023-42795</cve>
+    <cve>CVE-2023-45648</cve>
+  </suppress>
+  <suppress>
+    <notes><![CDATA[
+     jose4j-0.5.5.jar ]]></notes>
+    <cve>CVE-2023-31582</cve>
+  </suppress>
+  <suppress>
+    <notes><![CDATA[
+     tomcat-embed-websocket-10.1.12.jar ]]></notes>
+    <cve>CVE-2023-44487</cve>
+    <cve>CVE-2023-42795</cve>
+    <cve>CVE-2023-45648</cve>
+  </suppress>
+  <suppress>
+    <notes><![CDATA[
+     json-20230227.jar ]]></notes>
+    <cve>CVE-2023-5072</cve>
+  </suppress>
```

Comment:
is this needed? can we keep it sync with master?

Reply:
At the time of developement it wasnt available, but now i have synced with master, however we need add jose4j-0.5.5.jar